### ISSUE TITLE
List the plugin to Adobe Exchange?

### ISSUE THREAD

**Issue by YoungCatChen** [milch/LRExportHEIC#4]
This is a wonderful plugin and it deserves more discoverability. Do you want to list it to Adobe Exchange, so that other Lightroom users would be able to find it more easily?

Or... do you mind if I list it under my name as a free plugin, with mention of your name?

I'm looking forward to your reply.

**Comment by milch** [comment]
I consciously didn't publish it to the Adobe Exchange since 0 testing has gone into it other than on my machine, I have no clue about publishing on Adobe Exchange, and since I know it won't work on Windows at all. Does Adobe have a way to restrict plugins by platform?

**Comment by YoungCatChen** [comment]
I have no idea either. I tried, but because of the large amount of blanks to fill in, I'd rather wait for your approval. Do you mind if I keep trying to submit the plugin and, if success, adding you as an co-owner?

**Comment by milch** [comment]
Yup, no problem! Thanks for doing this.

**Comment by milch** [comment]
I see you managed to publish it here: https://exchange.adobe.com/apps/cc/108244/export-heic-heif

**Comment by YoungCatChen** [comment]
Yep! Finally managed to have published that. I still have some remaining tasks, like polishing settings etc., but I can prioritize adding you as a co-owner. I'm currently stuck in finding the right place to create an organization, and figure out if there is a need to create an organization in the first place. 

I feel like we can chat and/or meet virtually. If you don't mind, my anonymized email is kicker.untold_3p@icloud.com. I will reply you with my true email once you send me an email.
